wagtail/admin/views/editing_sessions.py

```diff
@@ -123,6 +123,28 @@ def ping(request, app_label, model_name, object_id, session_id):
                 if newest_revision.created_at > session_info["last_seen_at"]:
                     session_info["last_seen_at"] = newest_revision.created_at
 
+    # Sort the other sessions so that they are presented in the following order:
+    # 1. Prioritise any session with the latest revision. Then,
+    # 2. Prioritise any session that is currently editing. Then,
+    # 3. Prioritise any session with the smallest id, so that new sessions are
+    #    appended to the end of the list (they're shown last). We are not sorting
+    #    by last_seen_at to avoid shifting the order of the sessions as they
+    #    ping the server.
+    other_sessions = sorted(
+        other_sessions_lookup.values(),
+        key=lambda other_session: other_session["session_id"] or 0,
+    )
+    other_sessions = sorted(
+        other_sessions,
+        key=lambda other_session: other_session["is_editing"],
+        reverse=True,
+    )
+    other_sessions = sorted(
+        other_sessions,
+        key=lambda other_session: other_session["revision_id"] or 0,
+        reverse=True,
+    )
```

Comment:
We need to sort the `revision_id` and `is_editing` in descending order, but `session_id` in ascending order, so I followed the [multi-step technique from Python docs](https://docs.python.org/3/howto/sorting.html#sort-stability-and-complex-sorts). But I suppose since there can only be one session (at most) with `revision_id`, we only care about the presence and not the value itself, so we can coerce this to a boolean and negate both values to sort in descending order.

I've addressed this in [`714dc6f` (#12109)](https://github.com/wagtail/wagtail/pull/12109/commits/714dc6f0729cfb200d230667dcd4e54e6a74ab56) (along with the other comment for the `if` check).